docs/source/concepts.rst

```diff
@@ -30,9 +30,9 @@ Natural Language Understanding (NLU)
 ------------------------------------
 
 NLU is responsible for obtaining a structured representation of raw text utterances.
-Currently, it entails intent classification, entity recognition, and user satisfaction prediction.
+Currently, it entails dialogue acts recognition, key-value annotation, and user satisfaction prediction.
```

Comment:
Nit: "key-value annotation" => "slot-value annotations"? Or is this referring to something else?

Reply:
It refers to something else such as sentiment annotation. Note that we have generic annotation as key-value (class Annotation) and the specific slot-value annotation (class SlotValueAnnotation).